assets/css/ui.dialog.scss

```
@@ -6,6 +6,10 @@
 	position: static;
 }
 
+.jm-dialog:not([open]) {
+	display: none!important;
```

Could we add an space here just in case?

```suggestion
	display: none !important;
```

`npm run lint:css` was also complaining about this: `  10:15  ✖  Expected single space before "!"                                                 declaration-bang-space-before`

Also, it mentioned that this [selector is duplicated here](https://github.com/Automattic/WP-Job-Manager/pull/2841/files#diff-6a4f6e3811f7a7104c5095fb43d757444f0ee296d0b019f283b2b23c6d109b38R114-R116). Maybe worth merging the declarations?

Also, WDYT adding a comment to document that this is related to an issue on safari?